commit 3c2c926823305139ef7867dcffa76b3a505a3b02
Author: Greg Kroah-Hartman <gregkh@suse.de>
Date:   Tue Oct 12 13:20:38 2010 -0700

    Staging: brcm80211: util: remove unneeded use of uintptr
    
    uintptr shouldn't be used when you are just casting to a pointer anyway,
    so remove it from these usages.
    
    Cc: Brett Rudley <brudley@broadcom.com>
    Cc: Henry Ptasinski <henryp@broadcom.com>
    Cc: Nohee Ko <noheek@broadcom.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

--- a/drivers/staging/brcm80211/util/aiutils.c
+++ b/drivers/staging/brcm80211/util/aiutils.c
@@ -134,7 +134,7 @@ void ai_scan(si_t *sih, void *regs, uint
 	case SPI_BUS:
 	case SDIO_BUS:
 #endif				/* BCMSDIO */
-		eromptr = (u32 *) (uintptr) erombase;
+		eromptr = (u32 *)erombase;
 		break;
 
 	default:
@@ -358,8 +358,8 @@ void *ai_setcoreidx(si_t *sih, uint core
 	case SPI_BUS:
 	case SDIO_BUS:
 #endif				/* BCMSDIO */
-		sii->curmap = regs = (void *)((uintptr) addr);
-		sii->curwrap = (void *)((uintptr) wrap);
+		sii->curmap = regs = (void *)addr;
+		sii->curwrap = (void *)wrap;
 		break;
 
 	default:
